x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -1194,6 +1194,38 @@ a:keyword           | upper:keyword         | lower:keyword
 π/2 + a + B + Λ ºC  | Π/2 + A + B + Λ ºC    | π/2 + a + b + λ ºc
 ;
 
+reverse
+required_capability: fn_reverse
+from employees | sort emp_no | eval name_reversed = REVERSE(first_name) | keep emp_no, first_name, name_reversed | limit 1;
+
+emp_no:integer  | first_name:keyword    | name_reversed:keyword
+10001           | Georgi                | igroeG
+;
+
+reverseRow
+required_capability: fn_reverse
+// tag::reverse[]
+ROW message = "Some Text" | EVAL message_reversed = REVERSE(message);
+// end::reverse[]
+
+// tag::reverse-result[]
+message:keyword | message_reversed:keyword
+Some Text       | txeT emoS
+// end::reverse-result[]
+;
+
+reverseEmoji
+required_capability: fn_reverse
+// tag::reverseEmoji[]
+ROW message = "😂😁😅😐🥺😢😭" | EVAL message_reversed = REVERSE(message);
+// end::reverseEmoji[]
+
+// tag::reverseEmoji-result[]
+message:keyword | message_reversed:keyword
+😂😁😅😐🥺😢😭 | 😭😢🥺😐😅😁😂
+// end::reverseEmoji-result[]
```

Comment:
Nice - small comment, I would use different emoticons to better signal visually the reversal: 🏡🚌✈ becomes ✈ 🚌🏡

Reply:
Additionally add also a string that combines and intertwines regular chars + emoticons + extended ones (こんにちは) and check they are properly reversed (namely that the encoding is preserved).

Reply:
> Nice - small comment, I would use different emoticons to better signal visually the reversal

I think of the current ones like a sort of easter egg in the docs... you have to squint a bit and then you're rewarded with "getting it."

That said, I defer to you — LMK

cc @leemthompo in case he has input

Reply:
I think Costin's right that it would be easier with a bit more visual variance in the emojis but we're knee deep in nit territory here 😆

Reply:
Actually Liam this is the most important conversation going on in this PR right now 😆

Okay, I'll take another stab at emoji art 🖼️

Reply:
My battle for these emojis is met with opposition at every turn... first it was our handling of unicode graphemes and now it's a custom font on the docs website... will I prevail? Not sure at all

https://github.com/user-attachments/assets/f77d5857-fb2c-4fea-aea2-7047eb86bbed